.gitignore

```diff
@@ -9,11 +9,12 @@
 *.map
 config.log
 /retroarch
-/retroarch.*
+/retroarch.cfg
+/retroarch.exe
```

Comment:
.* covers .cfg and .exe... Could ! the ones that you actually want in there. Same goes for joyconfig.